src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Shape {
+    private double height;
+    private double width;
+
+    public Rectangle(String color, double height, double width) {
+        super(Colors.valueOf(color));
+        this.height = Math.round(Math.abs(height));
+        this.width = Math.round(Math.abs(width));
```

Comment:
Rounding the dimensions using `Math.round()` might not be necessary unless specified by the task requirements. Ensure this aligns with the intended functionality.